ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Query.hs

```diff
@@ -59,7 +69,6 @@ implGetSnapshotFor mpEnv pt slot ticked = do
                        (ForgeInKnownSlot slot ticked)
     MempoolEnv { mpEnvStateVar         = istate
                , mpEnvLedgerCfg        = cfg
-               , mpEnvLedger           = ldgrInterface
```

Comment:
Maybe a stupid question: if the snapshotting doesn't use the `LedgerInterface` anymore, should the other mempool functions do the same?